### PR TITLE
Add propless handler

### PR DESCRIPTION
**Pull request by eoliphan**

### CODE REVIEW

File: src/fulcro/incubator/ui/core.cljs [hunk 1]
```diff
@@ -14,6 +15,29 @@
   "Wrap a react component with localized css support (like on dom/*)"
   [component]
   (fn localized-factory
+    ([]
```
Comment (awkay) [inline src/fulcro/incubator/ui/core.cljs]: I think you got it backwards.  Localized means it should use the localized-dom, and not the plain dom :)

File: src/fulcro/incubator/ui/core.cljs [hunk 2]
```diff
@@ -14,6 +15,29 @@
...
+    ([arg arg2 arg3]
+     (if (keyword? arg)
+       (c-dom/macro-create-element component [arg2 arg3] arg)
+       (c-dom/macro-create-element component [arg arg2 arg3])))
+    ([arg arg2 arg3 & rest]
+     (if (keyword? arg)
+       (c-dom/macro-create-element component (concat [arg2 arg3] rest) arg)
+       (c-dom/macro-create-element component (concat [arg arg2 arg3] rest))))))
+
+(defn component-factory-nonlocalized
```
Comment (awkay) [inline src/fulcro/incubator/ui/core.cljs]: Let's call this one `component-factory` I think

**Comment by awkay** [comment]
Also, please confirm that you installed this locally (e.g. as a SNAPSHOT version with `lein install`) and tested that it works the way you expect.

File: src/fulcro/incubator/ui/core.cljs (hunk 1 above)
Comment (eoliphan) [reply src/fulcro/incubator/ui/core.cljs]: ah heck... i sure did lol

**Pull request merged**